Entities/AHuman.cpp

```diff
@@ -3163,24 +3163,18 @@ void AHuman::Update()
 			m_JetTimeLeft = std::min(m_JetTimeLeft + g_TimerMan.GetDeltaTimeMS() * 2.0F, m_JetTimeTotal);
 		}
 
-		// If pie menu is on, keep the angle to what it was before
-		if (m_Controller.IsState(PIE_MENU_ACTIVE))
-		{
-			// Don't change anything
-		}
-		// Direct the jetpack nozzle according to movement stick if analog input is present
-		else if (m_Controller.GetAnalogMove().GetMagnitude() > 0.1F)
-		{
-			//To-do: test whether this works properly
-			float jetAngle = (m_Controller.GetAnalogMove().GetAbsRadAngle() - c_PI);
-			if (jetAngle > c_PI) { jetAngle -= c_TwoPI; }
-			m_pJetpack->SetEmitAngle(FacingAngle(jetAngle * m_JetAngleRange));
-		}
-		// Or just use the aim angle if we're getting digital input
-		else {
+		float maxAngle = c_HalfPI * m_JetAngleRange;
+		// If pie menu is on, keep the angle to what it was before.
+		if (m_Controller.IsState(PIE_MENU_ACTIVE)) {
```

Comment:
I don't really like the empty if thing.
I'd rather you do
```cpp
if (!m_Controller.IsState(PIE_MENU_ACTIVE)) {
	if (m_Controller.GetAnalogMove().GetMagnitude() > 0.1F) {
		...
	} else {
		...
	}
}
```


Applies to ACrab.cpp as well.